test/jdk/java/nio/channels/Channels/TransferTo.java

```diff
@@ -282,4 +280,29 @@ public OutputStream output(Consumer<Supplier<byte[]>> spy) throws Exception {
         };
     }
 
+    @BeforeClass
+    public static void createTempFileRegistry() {
+        tempFiles = new LinkedList();
```

Comment:
This could be initialized as part of the declaration.

Reply:
Yes I could do that but I think it is better to understand for the reader as it clearly explains that this code is part of the test management, not part of the test logic.

Reply:
This is a simple test so really no need  not do the initialization as part of the declaration given there is no other work done by this method.

Reply:
As I explained, this is not written due to a *need* but due to *deliberate separation of concerns*. So is separation of concerns a design strategy that is forbidden in OpenJDK?

Reply:
It's better to avoid using raw-types unless there's a specific reason to do so (I guess this is just an oversight).

Reply:
Yes this was just an oversight, and this thread got outdated due to Alan`s resolution.